src/command/pack.rs

```diff
@@ -9,7 +9,15 @@ pub fn pack(path: Option<String>, log: &Logger) -> result::Result<(), Error> {
     let crate_path = set_crate_path(path);
 
     info!(&log, "Packing up the npm package...");
-    npm::npm_pack(&crate_path)?;
+    match npm::npm_pack(&crate_path) {
+        Ok(r) => r,
+        Err(Error::Io { .. }) => {
+            return Err(Error::DirNotFound {
+                message: "Unable to find the pkg directory".to_owned(),
+            });
+        }
+        Err(e) => return Err(e),
```

Comment:
These explicit `return` statements feel like code smells. Maybe I can use the `?` operator here?

Reply:
I should try something like this instead:

```rust
my_function_call().map_err(|e| match {
  interesting_error => something_else,
  stuff => stuff,
})?
```